conf/upf.json

```diff
@@ -134,6 +134,7 @@
 
     "": "Control plane controller settings",
     "cpiface": {
+        "": "peers: ['148.162.12.214']",
```

Comment:
Please use:

```
"peers": ['148.162.12.214'],
```

Reply:
Temporarily commented because it is probably not supported in CI Build.

Reply:
I suppose that it CI will pass if you get rid of exiting with `fatal`, when peers are not provided.

Reply:
We cannot merge this PR with a commented parameter, as it introduces confusion. However, once you got rid of `fatal` log, it should be safe to uncomment it.

Reply:
Done